packages/trader/src/Modules/SmartChart/Components/buy-toast-notification.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { MobileWrapper, Toast, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { findContractCategory } from '../../Trading/Helpers/contract-type';
+import { observer, useStore } from '@deriv/stores';
+
+type TContracType = {
+    text: string;
+    value: string;
+};
+
+type TBuyToastNotificationProps = {
+    portal_id?: string;
+};
+
+const BuyToastNotification = observer(
+    ({ portal_id = 'popup_root' }: React.PropsWithChildren<TBuyToastNotificationProps>) => {
```

Comment:
```suggestion
const BuyToastNotification = observer(() => {
```
@hamza-deriv The component doesn't accept any children, So there is no need to use `PropsWithChildren`, Also I think the `portal_id` prop is redundant and we can remove it from props.